plugins/woocommerce/tests/e2e-pw/tests/smoke-tests/update-woocommerce.spec.js

```diff
@@ -7,115 +8,123 @@ const {
 	deleteZip,
 } = require( '../../utils/plugin-utils' );
 
-const pluginZipPath = path.resolve( __dirname, '../../tmp/woocommerce.zip' );
+let pluginZipPath;
 
-test.describe( 'WooCommerce plugin can be uploaded and activated', () => {
-	// Skip test if UPDATE_WC is falsy.
-	test.skip(
-		! Boolean( UPDATE_WC ),
-		`Skipping this test because UPDATE_WC is falsy: ${ UPDATE_WC }`
-	);
+test.describe.serial(
+	'WooCommerce plugin can be uploaded and activated',
+	() => {
+		// Skip test if UPDATE_WC is falsy.
+		test.skip(
+			! Boolean( UPDATE_WC ),
+			`Skipping this test because UPDATE_WC is falsy: ${ UPDATE_WC }`
+		);
```

Comment:
I made this test runnable only when `UPDATE_WC` has a truthy value. So that this test will be skipped on PR's and in other workflows that won't necessarily need to update the WooCommerce version.